modules/space/translation/gptranslation.cpp

```diff
@@ -55,7 +55,7 @@ namespace {
 namespace openspace {
 
 documentation::Documentation GPTranslation::Documentation() {
-    return codegen::doc<Parameters>("space_transform_gp");
+    return codegen::doc<Parameters>("space_translation_gp");
```

Comment:
Note: changed key => should always be checked in documentation and codegen links. Doubt that we've referrred to this anywhere though

Reply:
Agreed and I did check